cloudfoundry-client-spring/src/test/java/org/cloudfoundry/spring/uaa/identityzonemanagement/SpringIdentityZoneManagementTest.java

```diff
@@ -47,7 +53,7 @@ protected RequestContext getRequestContext() {
         @Override
         protected CreateIdentityZoneResponse getResponse() {
             return CreateIdentityZoneResponse.builder()
-                .createdAt("2015-07-27T22:43:20Z")
+                .createdAt(new Date(1426258488910L))
```

Comment:
The problem here is actually, that it shouldn't be date at all.  The type is a `Long` and while we know that it represents millis-since-epoch, the `-client` APIs are supposed to map as closely as possible to the original data types.  So the `CreateIdentityZoneResponse` needs to actually use `Long`s rather than `Date`s.